README.md

```diff
@@ -107,6 +108,29 @@ When `--token your-token` is provided, an authentication header is required. For
 ~$ curl localhost:8980/transactions -H "X-Indexer-API-Token: your-token"
 ```
 
+# Settings
+
+Settings can be provided from the command line, a configuration file, or an environment variable
+
+### daemon mode
+
+```
+./algorand-indexer daemon [options]
+```
+
+| Command Line Flag (long) | (short) | Config File                | Environment Variable               |
+| ------------------------ | ------- | -------------------------- | ---------------------------------- |
+| postgres                 | P       | postgres-conenction-string | INDEXER_POSTGRES_CONNECTION_STRING |
```

Comment:
I'm looking at this and viper.RegisterAlias() and util.BindFlags() and wondering if f.Name might still just be "postgres" and this might build "INDEXER_POSTGRES"

Reply:
You're right, this worked for the config file but isn't working properly for the environment variable. I'll do some debugging.

Reply:
The bug I saw was something else. For some reason viper doesn't swap - and _ automatically so I needed `viper.SetEnvKeyReplacer(strings.NewReplacer("-", "_"))` as part of the config.

You're still correct that in Bindflags f.Name is the cobra name (since the alias isn't part of cobra). This code works because the alias isn't resolved until the value is looked up, in this case `viper.IsSet`. So `viper.IsSet("postgres")` and `viper.IsSet("postgres-connection-string")` both fetch the same value.